File: Selenium/Activities/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

WDService = webdriver.FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=WDService) as driver:
    driver.get("https://v1.training-support.net/selenium/target-practice")
    print("Home Page title is :",driver.title)
    ThirdHeadr =driver.find_element(By.XPATH,"//h3[@id = 'third-header']").text
    print("Thrird header is :" +ThirdHeadr)
    color = driver.find_element(By.XPATH,"//h5[text() = 'Fifth header']").value_of_css_property("color")
    print("Fifth header color is :" +color);
    violet_color = driver.find_element(By.XPATH,"//button[text()='Violet']").get_dom_attribute("class")
    print("Violet Color is :" +violet_color);
    Grey_Text = driver.find_element(By.XPATH,"//button[text() = 'Grey']").text
    print("Grey Text is :" +Grey_Text);
    driver.close()

File: Selenium/Activities/Activity4.java
package activities;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity4 {
    public static void main(String[] args){
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/target-practice");
        System.out.println("Home Page title is :" + driver.getTitle());
       String ThirdHeadr = driver.findElement(By.xpath("//h3[@id = 'third-header']")).getText();
       System.out.println("Thrird header is :" +ThirdHeadr);
       String color = driver.findElement(By.xpath("//h5[text() = 'Fifth header']")).getCssValue("color");
       System.out.println("Fifth header color is :" +color);
       String violet_color =driver.findElement(By.xpath("//button[text()='Violet']")).getAttribute("class");
       System.out.println("Violet Color is :" +violet_color);
       String Grey_Text = driver.findElement(By.xpath("//button[text() = 'Grey']")).getText();
       System.out.println("Grey Text is :" +Grey_Text);
       driver.close();

    }
}
